src/Sarif/CodeGenHints.json

```diff
@@ -146,11 +146,6 @@
       "kind": "DictionaryHint"
     }
   ],
-  "Invocation.ResponseFiles": [
-    {
-      "kind": "DictionaryHint"
-    }
```

Comment:
>} [](start = 4, length = 1)

This property is not a dictionary. `DictionaryHint` was (correctly) ignored.

Reply:
From the property description, it sounds like this is in fact a dictionary. What am I missing?

The property names in this object SHALL contain the names of all the environment variables in the tool's execution environment. The value of each property SHALL be a string containing the value of the specified environment variable. If the value of the environment variable is an empty string, the corresponding property value SHALL be an empty string.

---
In reply to: [217861968](https://github.com/Microsoft/sarif-sdk/pull/1089#discussion_r217861968) [](ancestors = 217861968)

Reply:
As we discussed: `invocation.responseFiles` is (appropriately) a list of `fileLocation`. Since it is (correctly) not declared as an `object` in the schema, this `DictionaryHint` is ignored.

I think you are reviewing the wrong PR. You already approved this one and it is merged. Please take a look at #1091. That's the one currently in review.